infrastructure/build/fab/site_specific/default/setup_cray.py

```diff
@@ -108,6 +108,8 @@ def setup_cray(build_config: BuildConfig, args: argparse.Namespace) -> None:
     linker.add_lib_flags("yaxt", ["-lyaxt", "-lyaxt_c"])
     linker.add_lib_flags("xios", ["-lxios"])
     linker.add_lib_flags("hdf5", ["-lhdf5"])
+    linker.add_lib_flags("shumlib", ["-lshum"])
```

Comment:
That repeated stuff is pretty annoying, but I can't see an easy way around that - short of adding a function that adds all default libraries for an application (i.e. exactly that code). So that function would just get called from all setup_XX functions. No need to fix this now, would be out of scope.